src/components/listbox/listbox.jsx

```diff
@@ -123,6 +124,7 @@ Listbox.propTypes = {
   children: PropTypes.node,
   className: PropTypes.string,
   labelledBy: PropTypes.string.isRequired,
+  ariaExpanded: PropTypes.bool,
```

Comment:
I think we shoudn't highlight the "aria" about this prop, instead favoring a more "standard" phrasing. I think this would encourage others to hook more "normal props" to their aria equivalents. i.e. let's rename `ariaExanded` to `expanded`.

Reply:
I can buy it, done!